dojo/tools/aqua/parser.py

```diff
@@ -100,4 +95,41 @@ def get_item(resource, vuln, test):
         component_version=resource.get('version'),
         impact=severity)
 
-    return finding
+
+def get_item_v2(item, test):
+    cve = item['name']
+    file_path = item['file']
+    url = item.get('url')
+    severity = severity_of(float(item['score']))
+    description = item.get('description')
+    solution = item.get('solution')
+    fix_version = item.get('fix_version')
+    if solution:
+        mitigation = solution
+    elif fix_version:
+        mitigation = f'Upgrade to {fix_version}'
+    else:
+        mitigation = 'No known mitigation'
+
+    return Finding(title=f'{cve}: {file_path}',
```

Comment:
It seems that `name` would contain what would be considered the `file_path` or `component_name` in DefectDojo, e.g. for java components.